lib/utils/azure_storage.ts

```diff
@@ -0,0 +1,48 @@
+import * as azureStorage from "azure-storage";
+import { Option, option } from "ts-option";
+import { Either, left, right } from "./either";
+
+// Storage connection string may be process.env.AzureWebJobsStorage;
+
+export type BlobService = azureStorage.BlobService;
```

Comment:
I'm not very fond of aliasing and re-exporting a type from another library, is this strictly required or it is just to save an `import`?

Reply:
short answer: yes, that's exactly to save an import.

long answer: the original intent of this file was to abstract away some ops on storage (that's why was called just 'storage' and not 'azure_storage'). so these type aliases and the helper methods were needed to encapsulate "swappable" types and behaviors.

not that the semantic has changed and it's bound to azure, i agree that we can get rid of type aliases and helper methods.